discovery-provider/alembic/versions/e2a8aea2e2e1_add_aggregate_user_table.py

```diff
@@ -0,0 +1,395 @@
+"""Add aggregate user table
+
+Revision ID: e2a8aea2e2e1
+Revises: 92571f94989a
+Create Date: 2021-10-14 19:38:41.776025
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e2a8aea2e2e1"
+down_revision = "92571f94989a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("CREATE TABLE aggregate_user_table AS TABLE aggregate_user")
+    op.execute("ALTER TABLE aggregate_user_table ADD PRIMARY KEY (user_id)")
+    op.execute("DROP MATERIALIZED VIEW aggregate_user CASCADE")
+
+    op.execute("ALTER TABLE aggregate_user_table RENAME TO aggregate_user")
+
+    op.execute(
+        """
+        DROP MATERIALIZED VIEW IF EXISTS trending_params;
+
+        CREATE MATERIALIZED VIEW trending_params as
```

Comment:
is this because the cascade wipes these?

Reply:
Yeah and we have to cascade because we can't drop the materialized view while others depend on it. Now I'm thinking we should just explicitly drop the dependencies to make it more clear. I'll make that change.